docs/1-tendermint-overview.md

```diff
@@ -0,0 +1,92 @@
+
+These conditions ensure both the safety and liveness properties of the consensus algorithm because a validator cannot send a Precommit message without sufficient evidence and cannot send a Precommit message for a different value at the same time.
+
+Tendermint is notable for its simplicity and performance and is currently one of the best consensus algorithm used in production.
+
+## Properties of consensus 
+
+The Tendermint consensus algorithm guarantees the following properties:
+
+- **Agreement**: No two correct processes decide on different values
+
+- **Validity**: A value decided is a valid value, respect to the specifications of validity of a message
+
+- **Termination**: All correct processes eventually decide
+
+- **Integrity**: No correct process decides more than once in a single height 
+
+We limit the discussion to the agreement property only because its proof can be useful to better understand the fork accountability problem. 
+We invite interested readers to read the official paper for a more formal, complete and rigorous proof of all the properties listed above.
+
+## Agreement property
+
+In this section we'll give a simple proof of the agreement property for the Tendermint algorithm and show why the majority assumption is important for ensuring this property.
+
+The key idea behind the proof of the agreement property is that any two sets of *2f + 1* processes have at least one correct process in common respect to the Tendermint context.
+
+We know that Tendermint assumes that *n > 3f*, for simplicity we can simplify it such that:
+ 
+    n = 3f + 1
+
+We have two sets of *2f + 1* processes, their sum can be written as:
+ 
+    2(2f + 1) = 4f + 2 = 3f + 1 + f + 1 = n + f + 1
+     
+That means that the intersection of these two sets contain at least *f + 1* processes, therefore at least one correct process.  
+
+Given this result, it's quite easy to understand that, if there are at most *f* faulty processes, it's not possible to violate the agreement property in the Tendermint consensus algorithm, as we'll show in the next section.
+
+### Idea behind the proof
+
+The idea behind the proof is that it can never happen that there are two sets of *2f + 1* Prevote or Precommit messages in the same round *r* for different values. If that would be the case, we'll have at least *f + 1* processes that sent both the messages, which would mean *f + 1* are faulty. But at most *f* processes are faulty, so this is impossible.
+Moreover, at most one value can be locked in a round and if a correct process decided value *v* in round *r*, *v* will be locked for all the next rounds after *r* on that height and correct processes will not send Prevote/Precommit message for values different than *v*. Therefore, it's clear that *v* will be the only value that could be decided from that moment onwards.
```

Comment:
Not true that correct processes will not send PREVOTE for V'. However, at most 2f processes could do that, hence no quorum of them which implies that "nothing bad" will not happen.
Revise this please.

Reply:
You're right, fixed.